src/main/java/net/playeranalytics/extension/CMI/CMIExtensionFactory.java

```diff
@@ -20,32 +20,54 @@ of this software and associated documentation files(the "Software"), to deal
     OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
     THE SOFTWARE.
 */
-package net.playeranalytics.extension.pluginname;
+package net.playeranalytics.extension.CMI;
```

Comment:
package names should be lowercase

Reply:
Fixed with 2758a63b57e27edccb308a8853e29a89e1d06376